src/providers/DappProvider/helpers/login/login.ts

```diff
@@ -30,19 +30,26 @@ async function loginWithoutNativeToken(provider: IProvider) {
   };
 }
 
-async function loginWithNativeToken(
-  provider: IProvider,
-  nativeAuthConfig: NativeAuthConfigType
-) {
+async function loginWithNativeToken({
+  provider,
+  nativeAuthConfig,
+  token
+}: {
+  provider: IProvider;
+  nativeAuthConfig: NativeAuthConfigType;
+  token?: string;
+}) {
   const {
     network: { apiAddress }
   } = getNetworkConfig();
 
   const nativeAuthClient = nativeAuth(nativeAuthConfig);
 
-  const loginToken = await nativeAuthClient.initialize({
-    noCache: true
-  });
+  const loginToken =
```

Comment:
maybe nicer 
token = '' in props
then
let loginToken = token
then
if (!loginToken) {
loginToken = await..